lib/digital_public_works/posts.ex

```diff
@@ -51,10 +57,11 @@ defmodule DigitalPublicWorks.Posts do
       {:error, %Ecto.Changeset{}}
 
   """
```

Comment:
@nickgartmann what is a good way to handle passing multiple associated structs to a context method? Both my current solution, and the previous solution of passing positional arguments feels a bit wrong